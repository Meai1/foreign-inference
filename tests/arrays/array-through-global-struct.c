struct S {
  int * fld;
  double d;
};

extern int x;
struct S s;

int f(int *arr1, double ** arr2) {
  s.fld = arr1;
  s.d = arr2[6][x];
  return s.fld[x] / s.d;
}
