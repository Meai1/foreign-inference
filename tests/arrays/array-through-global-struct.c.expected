fromList [ (("@f", "%arr2"), 2)
         ]
